package.json

```diff
@@ -469,6 +486,10 @@
         {
           "command": "azure-iot-toolkit.deleteModule",
           "when": "never"
+        },
+        {
+          "command": "azure-iot-toolkit.invokeModuleDirectMethod",
```

Comment:
we will not show any module related command in palette?

Reply:
Based on BI data, it is low priority. I have created an issue to track it: https://github.com/Microsoft/vscode-azure-iot-toolkit/issues/160